index.html

```diff
@@ -1223,6 +1223,58 @@
 };//]]>
 
 </script>
+
+<script>
+function saveSVG () {
+  var markup = getSVG();
+  var doc = new DOMParser().parseFromString(markup, "text/xml");
+  var svg = doc.children[0];
+  var defs = svg.children[0].children[2];
+
+  appendStyles(doc, defs);
+
+  saveFile(svg);
+}
+
+function getSVG() {
+  var svg = document.getElementById("chart");
```

Comment:
The title and color palettes legend are outside the `svg#chart`.
It seems important to include the legend in any export, if possible.

Reply:
I guess that there is no ideal way to include those without re-implementing the legend directly in svg.
So in the meanwhile I'll document that the svg is only exporting the chart itself.